### PR TITLE
feat: Add itemLayout to change the item appearance  like iOS

### PR DESCRIPTION
**Pull request by finalpets**
Also add titleSize property and some test for item ItemLayoutOption.LIKE_IOS

### CODE REVIEW

File: whatsnew/src/main/res/layout/whatsnew_item_ios.xml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    android:orientation="horizontal"
+    android:paddingBottom="16dp"
+    android:id="@+id/root"
+    android:paddingTop="8dp">
+    <RelativeLayout
+        android:padding="8dp"
+        android:layout_gravity="center"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        >
```
Comment (TonnyL) [inline whatsnew/src/main/res/layout/whatsnew_item_ios.xml]: nit: code style
1. reformat code ✅
2. rearrange code ✅
3. optimize imports ✅
Screen Shot 2020-10-28 at 11 09 01 AM

File: whatsnew/src/main/java/io/github/tonnyl/whatsnew/util/ItemLayoutOption.kt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package io.github.tonnyl.whatsnew.util
+
+/**
+ * Created by lizhaotailang on 30/11/2017.
```
Comment (TonnyL) [inline whatsnew/src/main/java/io/github/tonnyl/whatsnew/util/ItemLayoutOption.kt]: 🤔

**Review by TonnyL — APPROVED** [review]
Nice work 👍!
And if code is reformatted it's even better!

File: whatsnew/src/main/java/io/github/tonnyl/whatsnew/util/ItemLayoutOption.kt (hunk 2 above)
Comment (finalpets) [reply whatsnew/src/main/java/io/github/tonnyl/whatsnew/util/ItemLayoutOption.kt]: Sorry i just duplicate the same class presentationOptions , can i put my name in the created class?

Comment (TonnyL) [reply whatsnew/src/main/java/io/github/tonnyl/whatsnew/util/ItemLayoutOption.kt]: Of course you can 🎉

**Review by TonnyL — APPROVED** [review]

**Pull request merged**